### PR TITLE
Release/ship v7.3.0

### PR DESCRIPTION
**Pull request by roblarsen**

### CODE REVIEW

**Comment by roblarsen** [comment]
👀

**Comment by coliff** [comment]
minor thing really, but I realised i had Prettier formatting turned on when editing the editorconfig and it changed all the bulletpoints from `*` to `-` . what do you think? worth changing it back or fine keeping it like it is in this PR?

**Comment by coliff** [comment]
In any case, I suggest to change the bullet:

> - HTTPS link to https://ogp.me/ ([#2155](https://github.com/h5bp/html5-boilerplate/pull/2155))

to:

> - Small docs maintenance updates ([#2155](https://github.com/h5bp/html5-boilerplate/pull/2155)), ([#2164](https://github.com/h5bp/html5-boilerplate/pull/2164)), ([#2165](https://github.com/h5bp/html5-boilerplate/pull/2165)), ([#2167](https://github.com/h5bp/html5-boilerplate/pull/2167)) & ([#2168](https://github.com/h5bp/html5-boilerplate/pull/2168))

**Comment by roblarsen** [comment]
It looks like we're using "*" in other places so that's fine.

**Comment by roblarsen** [comment]
small updates

**Comment by coliff** [comment]
I just saw that Modernizr v3.8.0 was released. Worth updating?

https://github.com/Modernizr/Modernizr/blob/v3.8.0/CHANGELOG.md

**Comment by coliff** [comment]
nice one on updating Modernizr 👍 .

On the `CHANGELOG.md`, did you search and replace `-` with `*`? There are some places where `-` make more sense. see the commit diff to see what I mean. 
https://github.com/h5bp/html5-boilerplate/pull/2169/files

File: README.md [hunk 1]
```diff
@@ -23,15 +23,19 @@ way that you want.
 
 Choose one of the following options:
 
-- Download the latest stable release from
-  [html5boilerplate.com](https://html5boilerplate.com/).
-- Clone the git repo — `git clone
+* Download the latest stable release from
+  [html5boilerplate.com](https://html5boilerplate.com/). This is a snapshot of the `dist` folder. On Windows, Mac and from the graphical file manager on Linux unzipping this folder will output to a folder named something like `html5-boilerplate_v7.3.0`From the command line will need to create a folder and unzip the contents into that folder.
```
Comment (coliff) [inline README.md]: suggest changing:

> `html5-boilerplate_v7.3.0`From the command line

to:

> `html5-boilerplate_v7.3.0`. From the command line

File: CHANGELOG.md [hunk 2]
```diff
@@ -180,11 +187,11 @@
 * Update to Apache Server Configs 1.1.0.
 * Add `initial-scale=1` to the viewport `meta`
   ([#1398](https://github.com/h5bp/html5-boilerplate/pull/1398)).
-* Vertical centering for audio-, canvas- and video-tags
+* Vertical centering for audio-, canvas* and video-tags
```
Comment (coliff) [inline CHANGELOG.md]: we need to revert the changed `-` to `*` in some places - like here.

Comment (roblarsen) [reply CHANGELOG.md]: 👍 I missed a few

**Comment by roblarsen** [comment]
@coliff I would like today to be the day, so if you have time for another once over.

**Comment by coliff** [comment]
cool - checking now... 

- For this final PR should the version number in `package.json` be 7.3.0 (and not 7.3.0-rc1)
- Not critical at all, but the line lengths in the markdown files vary a lot. We should properly sort this out at some point (not certain of the best way right now though)

**Comment by roblarsen** [comment]
> For this final PR should the version number in package.json be 7.3.0 (and not 7.3.0-rc1)

I actually made the change and never pushed it. pushing now.

**Comment by coliff** [comment]
just spotted something else... the html.md includes part of the HTML template with:
`<script src="js/vendor/modernizr-3.7.1.min.js"></script>`
this should now be:
`<script src="js/vendor/modernizr-3.8.0.min.js"></script>`

**Comment by coliff** [comment]
ugh, I'm finding more things... on `extend.md` we should change that Google+ / Schema.org section needs updating (as Google+ is dead). At the very least we should:
- Change that section title to just: Schema.org
- Remove 'While it helps you to control information displayed on Google+, '
- In that example HTML snippet change `http://schema.org/Article` to `https://schema.org/Article`   (REF: https://schema.org/docs/faq.html#19 )

**Comment by roblarsen** [comment]
good eye. I can make the changes after lunch.

**Comment by coliff** [comment]
a few last (optional) things I thought of:
- Could consider bumping min version of NPM in `package.json` to v10. v8 is in maintenance mode and will be unsupported in 2 months time. (https://nodejs.org/en/about/releases/)
- There's a few packages in package.json which could be bumped babel, eslint, eslint-plugin-mocha, glob and ssri.
- In the `.editorconfig` we have a rule to NOT trim trailing spaces in markdown. I don't know why that is set there - trailing whitespaces are annoying and I don't think ever needed. I'm not 100% certain on this though...

**Comment by roblarsen** [comment]
Back from vacation.

**Comment by coliff** [comment]
welcome back :-)  I'm around all day if I can help out with anything

**Comment by coliff** [comment]
lookin' good. Just update the date in the `CHANGELOG.md` for the new release I think.

**Comment by roblarsen** [comment]
the npm publish github action worked as advertised: https://www.npmjs.com/package/html5-boilerplate

**Pull request merged**